ui/app/AppLayouts/Wallet/popups/swap/SwapModal.qml

```diff
@@ -0,0 +1,55 @@
+import QtQuick 2.13
+import QtQuick.Layouts 1.15
+
+import utils 1.0
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Popups.Dialog 0.1
+
+StatusDialog {
+    id: root
+    title: qsTr("Swap")
+
+    // This should be the only property which should be used when being launched from elsewhere
+    property SwapFormData formData: SwapFormData {}
+
+    bottomPadding: 16
+    padding: 0
+
+    background: StatusDialogBackground {
```

Comment:
No need to override the `background` at all; just set `width` on the `StatusDialog`

Reply:
Or is it b/c of the color?

Reply:
yep exactly its white by default. Ive also not added header footer etc, should be done in the following tasks